cron/cron.ts

```diff
@@ -14,57 +14,59 @@ import cron from 'node-cron'
 import { handleSendMessageError } from '#helper/errorHandler'
 import { getHadith } from '#helper/getHadith'
 
-async function monthly() {
-  const now = new Date()
-  const currentMonth = now.getMonth() + 1
+async function yearly() {
   const keyboardMessage = HLanguage('region')
   const regions = Object.keys(keyboardMessage)
   const regionIds = Object.values(keyboardMessage)
   const daysOfWeek = ['Якшанба', 'Душанба', 'Сешанба', 'Чоршанба', 'Пайшанба', 'Жума', 'Шанба']
+  const prayTimes = []
 
   await Model.PrayTime.deleteMany()
 
-  for (let i = 0; i < regions.length; i++) {
-    const pdf = await axios.get(env.TIME_API + regionIds[i] + '/' + currentMonth, {
-      responseType: 'arraybuffer',
-    })
-    const pdfData = await pdfParser(pdf.data)
-    const data = pdfData.text.split('\n')
-    const prayTimes = []
-
-    for (let el of data) {
-      if (el.length > 20 && el.split(' ').length == 1) {
-        for (let day of daysOfWeek) {
-          if (el.includes(day)) {
-            let dayNumber = el.split(day)[0]
-            let times = el.split(day)[1].match(/.{1,5}/g) as RegExpMatchArray
-
-            prayTimes.push({
-              region: regions[i],
-              regionId: regionIds[i],
-              day: dayNumber,
-              fajr: times[0],
-              sunrise: times[1],
-              dhuhr: times[2],
-              asr: times[3],
-              maghrib: times[4],
-              isha: times[5],
-            })
+  for (let month = 1; month <= 12; month++) {
```

Comment:
hamma joyda month + 1 qilib ishlatmasdan shunchaki monthni 0 dan boshlasak qanday bo’ladi?

Reply:
<img width="691" alt="image" src="https://github.com/Xayrulloh/ReminderBot/assets/91729108/20031a85-cf65-4c15-9e54-e0768f9788c4">

as u can see we can't cause its starting from 1